DiscordBot.Modules/CommandModules/ReactionModule.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Modules.CommandModules
+{
+    public class ReactionModule : ModuleBase
+    {
+        private readonly ILogger<PingModule> logger;
+
+        public ReactionModule(ILogger<PingModule> logger)
+        {
+            this.logger = logger;
```

Comment:
Saw it right now: please rename to _logger

Reply:
unnecessary `this`: grrr

Reply:
that wasn't my fault, I copyed from you

Reply:
Lolwhut?
From where?

Reply:
PingModule from u

Reply:
@JKamsker @blaumeise20 fixed